bundles/org.openhab.binding.tradfri/README.md

```diff
@@ -65,17 +78,33 @@ The control outlet supports the `power` channel.
 
 A blind or curtain supports, beside `battery_level` and `battery_low` channels,  a `positon` channel.
 
+An air purifier supports:
+* a `fan_mode` and `fan_speed`, which allows for control and reading of the speed.
+* a `disable_led` and `lock_button`, to disabled the LED's or lock the button on the physical device.
+* a `air_quality_pm25` and `air_quality_rating`, which reads the particulate matter 2.5μm and corresponding indication of air quality.
+* a `filter_check_next` and `filter_check_alarm`, which represents the remaining number of minutes until next filter check and whether it is time to do the filter check now. Filter check must be completed through the TRÅDFRI app.
+* a `filter_uptime`, which represents the current time since last filter change.
+
 Refer to the matrix above.
 
-| Channel Type ID   | Item Type     | Description                                            |
-|-------------------|---------------|--------------------------------------------------------|
-| brightness        | Dimmer        | The brightness of the bulb in percent                  |
-| color_temperature | Dimmer        | color temperature from 0% = cold to 100% = warm        |
-| color             | Color         | full color                                             |
-| battery_level     | Number        | battery level (in %)                                   |
-| battery_low       | Switch        | battery low warning (<=10% = ON, >10% = OFF)           |
-| power             | Switch        | power switch                                           |
-| position          | Rollershutter | position of the blinds from 0% = open to 100% = closed |
+| Channel Type ID    | Item Type     | Description                                                                              |
+|--------------------|---------------|------------------------------------------------------------------------------------------|
+| brightness         | Dimmer        | The brightness of the bulb in percent                                                    |
+| color_temperature  | Dimmer        | color temperature from 0% = cold to 100% = warm                                          |
```

Comment:
Optional, just noticed it was also like this before:
```suggestion
| color_temperature  | Dimmer        | Color temperature from 0% = cold to 100% = warm                                          |
```

Reply:
On this one: the version of the README in the current head of the git displays like this:
![image](https://github.com/openhab/openhab-addons/assets/45715230/2887bfd0-e00a-4986-b380-75ce81dad770)

* lowercase "c" for "color temperature"
  * perhaps it has been capital "C" some time in the past, however this change is not a colateral of this PR 
* capital "t" for "The brightness" while all other texts in the table start (and are proposed to start for the new lines) with a lowercase letter.

So my proposal - unless there are project convetions for that - would be to align everything to lowercase in that column:
* no change on "color temperature" (keep it as in the head)
* lowercase first letter of all added descriptions
* change the capital "T" in "The brightness" (first line of the table) to lowercase "t"

Reply:
I'll opt in for uppercase as this seems to be how it was historically done in this add-on.

Reply:
Most important is to have consistency. But I would also opt for uppercasing first letter in the descriptions, since they are basically sentences.